resources/deployments/model/deployments.go

```diff
@@ -34,6 +34,7 @@ type DeploymentsModel struct {
 	deviceDeploymentLogsStorage DeviceDeploymentLogsStorage
 	imageLinker                 GetRequester
 	deviceDeploymentGenerator   Generator
+	imageContentType            string
 }
 
 func NewDeploymentModel(
```

Comment:
Not a huge problem, but the patch adds a 6th positional parameter to `NewDeploymentModel`. IMO it's a bit too many, even by Go standards. Do you think it could be replaced with a struct? e.g. 
```
type DeploymentModelConfig struct {
 	deploymentsStorage DeploymentsStorage
 	deviceDeploymentGenerator Generator
 	deviceDeploymentsStorage DeviceDeploymentStorage
 	deviceDeploymentLogsStorage DeviceDeploymentLogsStorage
 	imageLinker GetRequester
	imageContentType string
}

func NewDeploymentModel(config DeploymentModelConfig) {
...
}
```

@maciejmrowiec second opinion?